src/indev/lv_indev.c

```diff
@@ -1477,3 +1496,44 @@ static lv_result_t send_event(lv_event_code_t code, void * param)
 
     return LV_RESULT_OK;
 }
+
+static void indev_scroll_throw_anim_cb(void * var, int32_t v)
+{
+    LV_ASSERT_NULL(var);
+    LV_UNUSED(v);
+
+    lv_indev_t * indev = (lv_indev_t *)var;
+
+    _lv_indev_scroll_throw_handler(indev);
+
+    if(indev->pointer.scroll_dir == LV_DIR_NONE || indev->pointer.scroll_obj == NULL) {
+        if(indev->scroll_throw_anim) {
+            /*hacky*/
+            LV_LOG_INFO("stop animation");
+            lv_anim_set_time(indev->scroll_throw_anim, 0);
+        }
+    }
+}
+
+static void indev_scroll_throw_anim_ready_cb(lv_anim_t * anim)
+{
+    if(anim) {
+        indev_scroll_throw_anim_reset((lv_indev_t *)anim->var);
+    }
+}
+
+static void indev_scroll_throw_anim_start(lv_indev_t * indev)
+{
+    LV_ASSERT_NULL(indev);
+
+    lv_anim_t a;
+    lv_anim_init(&a);
+    lv_anim_set_var(&a, indev);
+    lv_anim_set_time(&a, 5000);
```

Comment:
If the duration is too large, under the current implementation mechanism of the animation, the callback will not be triggered when the value is the same, which will cause the animation frame to change too much. 

https://github.com/lvgl/lvgl/blob/423c288070ccfb90977529aae425b72419733c77/src/misc/lv_anim.c#L392-L396

We should ensure that the value of "value" changes with each frame during calculation, so that the scrolling animation is smooth. I updated the "value" configuration to better match a duration of 5000.

Reply:
We can make it the same as duration. This way it will change on each frame for sure.

Reply:
Currently, different curves in `lv_anim` use `lv_map` to round the value to the `LV_ANIM_RESOLUTION` range when calculating the value. Therefore, if the value is scaled beyond this range, it cannot be guaranteed that the value of each frame will be different from the previous value. Please see the relevant code below.

https://github.com/lvgl/lvgl/blob/224fb4370333a928088e83657fac32bce969c933/src/misc/lv_anim.c#L230-L233

https://github.com/lvgl/lvgl/blob/224fb4370333a928088e83657fac32bce969c933/src/misc/lv_anim.c#L22

Therefore, the solution of setting the value configuration to 1000000 is not feasible. In order to meet the demand for the maximum duration, I suggest configuring the value as the maximum value of `LV_ANIM_RESOLUTION`, and configuring the duration as the result of value*60. What do you think?

Reply:
Or maybe we should change the limit `LV_ANIM_RESOLUTION`.

Reply:
@kisvegabor What's your opinion?

Reply:
I think we are safe to set `LV_ANIM_RESOLUTION 65536`. And instead of 1000000 use 65535  in scroll animation.

Reply:
Ok, I have updated it.